### PR TITLE
Adding Saiyan Stuff

### PR DESCRIPTION
**Pull request by mcremone**

### CODE REVIEW

File: fnal_column_analysis_tools/arrays/Builder.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import awkward
```
Comment (nsmith-) [inline fnal_column_analysis_tools/arrays/Builder.py]: `from ..util import awkward`
This allows a global switch for the different awkward versions (`awkward.numba`, `awkward.cuda`, etc.)

File: fnal_column_analysis_tools/arrays/Builder.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import uproot, uproot_methods
```
Comment (nsmith-) [inline fnal_column_analysis_tools/arrays/Builder.py]: `uproot` isn't used, right?

File: tests/test_arrays.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
+import uproot, uproot_methods
+import numpy as np
+import fnal_column_analysis_tools
+from fnal_column_analysis_tools.arrays import Builder
+
+file = uproot.open("nano_5.root")
```
Comment (nsmith-) [inline tests/test_arrays.py:6]: Add this file to `tests/samples` if it isn't too big.  Also, I think the whole test should be wrapped in a descriptive function name.

**Review by nsmith- — CHANGES_REQUESTED** [review]

**Comment by lgray** [comment]
@mcremone Please implement the changes Nick has requested.

File: tests/test_arrays.py (hunk 3 above)
Comment (nsmith-) [reply tests/test_arrays.py:6]: Gotta change the path here too.  Also, 40mb is a bit big, possible to make it a bit smaller, say <1 mb?

Comment (mcremone) [reply tests/test_arrays.py:6]: Path is consistent no? I added the root file in tests.

**Comment by nsmith-** [comment]
Superseded by #67

**Pull request closed without merging**